compiler/src/dmd/frontend.h

```diff
@@ -7570,6 +7567,8 @@ extern bool isFuncHidden(ClassDeclaration* cd, FuncDeclaration* fd);
 
 extern bool isAbstract(ClassDeclaration* cd);
 
+extern void finalizeSize(AggregateDeclaration* ad);
+
 extern void lowerNonArrayAggregate(StaticForeach* sfe, Scope* sc);
 
 extern int32_t include(Condition* c, Scope* sc);
```

Comment:
I thought the consensus was to not introduce any more top-level c++ symbols, so that all these are here is worrying.